STMicroelectronics/STM32L4_L4+/samples/sprinkler.1/sprinkler_interface.c

```diff
@@ -107,17 +106,6 @@ static az_result sprinkler_1_stop_span_wrapper(az_span model_in_span, az_span* m
   return AZ_ULIB_TRY_RESULT;
 }
```

Comment:
`static az_result sprinkler_v1i1_end(void)` was used as a way to stop the sample on uninstall. is this functionality no longer necessary?

Reply:
It is necessary, but it is part of the business logic, not the interface, so I moved it to sprinkler.c.